test/runtests.py

```diff
@@ -187,11 +187,11 @@ def run_testlist(testlist):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument('--testlist', help='testlist file to execute', required=True)
+    parser.add_argument('testlists', help='testlist files to execute', nargs='+')
```

Comment:
This should be a required argument.

Reply:
It is. `nargs='+'` means require 1 or more.